starters/basic-starter/pages/[...slug].tsx

```diff
@@ -13,30 +13,84 @@ import { NodePerson } from 'components/node--person';
 import { NodePlace } from 'components/node--place';
 import { NodeBasicPage } from 'components/node--page';
 import { drupal } from '../lib/drupal';
+import { TaxonomyArticle } from '../components/taxonomy/taxonomy--article_type';
+import { TaxonomyPerson } from '../components/taxonomy/taxonomy--person_type';
+import { TaxonomyEvent } from '../components/taxonomy/taxonomy--event_type';
+import { TaxonomyPlace } from '../components/taxonomy/taxonomy--place_type';
 
 // List of all the entity types handled by this route.
-const CONTENT_TYPES = [
+const ENTITY_TYPES = [
   'node--page',
   'node--article',
   'node--event',
   'node--person',
   'node--place',
+  'taxonomy_term--article_type',
+  'taxonomy_term--categories',
+  'taxonomy_term--event_type',
+  'taxonomy_term--person_type',
+  'taxonomy_term--place_type',
 ];
 
-interface NodePageProps extends LayoutProps {
-  node: DrupalNode;
+interface EntityPageProps extends LayoutProps {
+  entity: DrupalNode | DrupalTaxonomyTerm;
+  additionalContent?: {
+    nodes?: DrupalNode[];
+  };
 }
 
-export default function NodePage({ node, menus }: NodePageProps) {
-  if (!node) return null;
-
+export default function EntityPage({
+  entity,
+  additionalContent,
+  menus,
+}: EntityPageProps) {
   return (
-    <Layout title={node.title} menus={menus}>
-      {node.type === 'node--page' && <NodeBasicPage node={node} />}
-      {node.type === 'node--article' && <NodeArticle node={node} />}
-      {node.type === 'node--event' && <NodeEvent node={node} />}
-      {node.type === 'node--person' && <NodePerson node={node} />}
-      {node.type === 'node--place' && <NodePlace node={node} />}
+    <Layout title={entity.title || entity.name} menus={menus}>
+      {entity.type === 'node--page' && (
+        <NodeBasicPage node={entity as DrupalNode} />
+      )}
+      {entity.type === 'node--article' && (
+        <NodeArticle node={entity as DrupalNode} />
+      )}
+      {entity.type === 'node--event' && (
+        <NodeEvent node={entity as DrupalNode} />
+      )}
+      {entity.type === 'node--person' && (
+        <NodePerson node={entity as DrupalNode} />
+      )}
+      {entity.type === 'node--place' && (
+        <NodePlace node={entity as DrupalNode} />
+      )}
+      {entity.type === 'taxonomy_term--article_type' && (
+        <TaxonomyArticle
+          additionalContent={additionalContent as { nodes: DrupalNode[] }}
+          taxonomy_term={entity as DrupalTaxonomyTerm}
+        />
+      )}
+      {entity.type === 'taxonomy_term--categories' && (
+        <TaxonomyArticle
```

Comment:
Should we explain with an inline comment, why are we rendering the categories with TaxonomyArticle? I'd be fine with removing support for categories since it makes the least sense without having support for including multiple content types

Reply:
I am fine with removing it too since it's a demo